modules/imaging_browser/php/Imaging_Session_ControlPanel.class.inc

```diff
@@ -96,13 +96,17 @@ class Imaging_Session_ControlPanel
             $params['PCandID'] = $timePoint->getCandID();
             $params['PVL']     = $timePoint->getVisitLabel();
         }
-        $tarchiveIDs = $DB->pselect(
-            "SELECT TarchiveID 
-            FROM tarchive 
+
+        // To enable DICOM download: Add pair: tarchiveID, ArchiveLocation
+        $qresult = $DB->pselectRow(
+            "SELECT TarchiveID, ArchiveLocation
+            FROM tarchive
             WHERE PatientName LIKE $ID",
```

Comment:
@christinerogers 

not sure how to use a prepared query with $ID being defined a few lines earlier with 2 cases and the second case being a concat of 3 different variables)...